app/models/question.rb

```diff
@@ -201,18 +201,20 @@ def annotations_per_org(org_id)
   # upon saving of question update conditions (via a delete and create) from params
   # the old_to_new_opts map allows us to rewrite the question_option ids which may be out of sync
   # after versioning
-  def update_conditions(param_conditions, old_to_new_opts, question_id_map)
+  def update_conditions(param_conditions, flash, old_to_new_opts, question_id_map)
+    res = true
     self.conditions.destroy_all
 
     if param_conditions.present?
       param_conditions.each do |_key, value|
-        saveCondition(value, old_to_new_opts, question_id_map)
+        res = saveCondition(value, flash, old_to_new_opts, question_id_map)
       end
     end
+    return res
```

Comment:
no need for the `return` here. Ruby methods always return the output of the last line.